### ISSUE TITLE
Training on a subset of classes (3 categories) in COCO data-set using yolov3-tiny

### ISSUE THREAD

**Issue by rohith14** [AlexeyAB/darknet#2574]
Couple of questions:

(1)I don't see a chart of loss vs iteration with '-map' flag during training

I use the following command (I have a display)
./darknet detector train coco/obj.data coco/yolov3-tiny_obj_anchorKmeans.cfg coco/yolov3-tiny.conv.15 -map

(2)
After training for about 100 epochs, the mAP is ~34%.  Is this expected?  I think it's on the lower end after training for about 100 epochs

Attachments for *.cfg and *.data
[yolov3-tiny_obj_anchorKmeans.txt](https://github.com/AlexeyAB/darknet/files/2952625/yolov3-tiny_obj_anchorKmeans.txt)
[obj.txt](https://github.com/AlexeyAB/darknet/files/2952627/obj.txt)
 
Thanks

**Comment by AlexeyAB** [comment]
1. Did you compile this repository https://github.com/AlexeyAB/darknet with `OPENCV=1` in the Makefile? 

2.  for yolov3-tiny.cfg the final mAP@0.5 should be ~33% on valid/test MS COCO dataset (if you tain for all 80 classes). So may be you can better accuracy for only 3 classes. If mAP increasing stops, then reduce learning_rate and continue training.

**Comment by rohith14** [comment]
1.  Sorry, I missed that.  I'll build with OpenCV.  Unfortunately, I have OpenCV 3.4.4 and runs into build issues.  I need OpenCV 3.4.0, i guess?

2. I will try it this out

Thanks
